### ISSUE TITLE
Custom Entry Slug doesn't work when View is embedded

### ISSUE THREAD

**Issue by rafaehlers** [GravityKit/GravityView#2207]
Here's my GravityView Settings:

![image](https://github.com/user-attachments/assets/1878028e-253b-4878-aaa0-21a85adb2492)

And the View Entry Slug settings:

![image](https://github.com/user-attachments/assets/0e4a3233-3ecc-465e-8e19-1aa0d3679cb5)

If the View is embedded into a page or on the home page, links to the Single Entry only use the Entry ID:

![image](https://github.com/user-attachments/assets/18e2152b-c8cc-4b32-838b-9310ff6ce2dc)

However, if we access the View custom post type directly, the custom entry slug works:

![image](https://github.com/user-attachments/assets/5aca42e8-acb6-4ffd-89a6-923097c9a8ed)

Here's a video: https://www.loom.com/share/4062a8d7f60946b29b23cf476b1c4108

Tested on GV 2.31.1

**Comment by zackkatz** [comment]
Just a long-shot: Did you try saving Permalinks settings

**Comment by rafaehlers** [comment]
@zackkatz that was the first thing I tried! hehe

**Comment by rafaehlers** [comment]
@doekenorg can you do this one next, please?

**Comment by doekenorg** [comment]
@rafaehlers this is a very tricky one. The problem is, that the hook to determine if we are using custom slugs does not contain the view (ID). Besides that, the setting is based on the actual View. When use the actual View page, we can determine the settings for that View, and make the changes for the Entry. But when you embed a View,  or maybe even multiple Views, on a single page; its not possible to determine the settings for that Entry. 

To summarize the problem: An Entry is not "connected" to a single View. So you need a way to determine the current View based on the page; which isn't possible (enough) when embedding. So it's not possible to retrieve the View setting for the Entry slug.
